SingularityService/src/main/docker/start.sh

```diff
@@ -15,7 +15,12 @@ DEFAULT_URI_BASE="http://${SINGULARITY_HOSTNAME:=localhost}:${SINGULARITY_PORT:=
 
 args+=( -Xmx${SINGULARITY_MAX_HEAP:-512m} )
 args+=( -Djava.net.preferIPv4Stack=true )
+args+=( -Ddw.server.applicationContextPath="${SINGULARITY_APP_CTX:=/singularity}" )
```

Comment:
we've got a few other environment variable floating around that serve a similar purpose (see: https://github.com/HubSpot/Singularity/blob/master/SingularityUI/gulpfile.js#L17) -- @ssalinas do you think it'd be good to consolidate here?